### PR TITLE
add test method to route facade

### PR DESCRIPTION
**Pull request by kkumar-gcc**
## 📑 Description

Closes https://github.com/goravel/goravel/issues/?

@coderabbitai summary

## ✅ Checks

- [ ] Added test cases for my code

### CODE REVIEW

File: route.go [hunk 1]
```diff
@@ -179,6 +179,10 @@ func (r *Route) Stop(ctx ...context.Context) error {
 	return nil
 }
 
+func (r *Route) Test(*http.Request) (*http.Response, error) {
+	panic("not support")
```
Comment (hwbrzzl) [inline route.go:183]: Can't implement this function?

**Review by hwbrzzl — DISMISSED** [review]
LGTM

File: route.go [hunk 2]
```diff
@@ -179,6 +180,14 @@ func (r *Route) Stop(ctx ...context.Context) error {
 	return nil
 }
 
+func (r *Route) Test(request *http.Request) (*http.Response, error) {
```
Comment (hwbrzzl) [inline route.go:183]: Could you add a simple test for it?

Comment (kkumar-gcc) [reply route.go:183]: Okay

**Review by hwbrzzl — APPROVED** [review]

**Pull request merged**